.happy/terraform/modules/wmg-batch/main.tf

```diff
@@ -0,0 +1,53 @@
+# This is a batch job
+# 
+
+data aws_region current {}
+
+data aws_caller_identity current {}
+
+resource aws_batch_job_definition batch_job_def {
+  type = "container"
+  name = "dp-${var.deployment_stage}-${var.custom_stack_name}-wmg-processing"
+  container_properties = <<EOF
+{
+  "jobRoleArn": "${var.batch_role_arn}",
+  "image": "${var.image}",
+  "memory": ${var.batch_container_memory_limit},
+  "environment": [
+    {
+      "name": "ARTIFACT_BUCKET",
```

Comment:
These two buckets are probably not needed. We should replace these with a new variable that points to a stable bucket that contains the WMG cube.

Reply:
I'm not sure I understand. We will need access to the artifact bucket to get the datasets that make up the cube. From the tech spec it looks like we need to create another bucket s3://wmg-$ENV

Reply:
Ok, I'll keep it then and add another variable for the WMG bucket. What about the cellxgene bucket?